tutorial/markdown/java/spring-boot.md

```diff
@@ -22,14 +22,50 @@ sdk_language:
 length: 30 Mins
 ---
 
-<!-- 
+<!--
   The name of this file does not need to be `tutorial-quickstart-java-springboot` because it is in the `tutorials/java/markdown` directory, so we can just call it `spring-boot`. The idea is that we can leave off `tutorial-quickstart` as a prefix.
 -->
 
 <!-- TODO:  Figure out how to add width to image size in try it now links -->
+
 [![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=java-springboot-quickstart-repo&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/java-springboot-quickstart)
 
-## Prerequisites
+## Table of Contents
```

Comment:
There are a few missing things:
* Data Model (Entities)
* Appendix
* Running the App
* Screenshots of the app